README.md

```diff
@@ -24,18 +24,38 @@ Here are the requirements for Magepack to work:
 3. If you are using Magento 2.3.3 or lower, you need `jquery.cookie` module shim [(patch provided and explained here)](https://github.com/magento/baler/issues/6).
 4. [Magepack Magento module](https://github.com/magesuite/magepack-magento) installed.
 
-Install with npm:
+Install (globally) with npm - requires admin privileges:
 
 ```
 npm install -g magepack
 ```
 
+Alternative: Install (locally) with npm:
+- If you don't have admin privileges and/or;
+- If installing globally is not an option available to you.
+
+```shell
+npm install magepack
+```
+The following commands allow you to download the essential x86_64 shared object files required by Chrome to operate in headless mode.
+
+Note: These files are designed specifically for x86_64 architecture.
+- I can provide the shared object files for other architectures upon request.
+
+```shell
+mkdir -p $HOME/.local/lib
+wget https://github.com/maximus-sallam/magepack/raw/master/x86_64/lib.tar.gz
```

Comment:
If you want these changes to be accepted here, you may like to update the URL to be *this* project instead of your fork thereof.

Reply:
My apologies. I'll fix this now.

Reply:
I have updated the URL to reflect your repo, however, since you are not hosting the shared object files, this URL will return a 404. I should note that these shared object files were the remaining files needed to get Chrome operational within our infrastructure, and not the total shared objects required. The files I've provided do not account for the required shared objects that were already installed within our infrastructure. For a total list of required shared object files, see the Dependencies list here: https://pptr.dev/troubleshooting